Client/Models/MapObject.cs

```diff
@@ -14,6 +14,7 @@
 using Library.SystemModels;
 using SlimDX;
 using SlimDX.Direct3D9;
+using static System.Windows.Forms.VisualStyles.VisualStyleElement.Window;
```

Comment:
this needs removing please - visual studio has a habit of auto adding incorrect using lines now

Reply:
Strange one, done thou!